<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1405</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="main_layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1405</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="title">
     <string>檔案 File</string>
    </property>
    <widget class="QMenu" name="menu_Load">
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="title">
      <string>載入 Load</string>
     </property>
     <addaction name="action_load_from_local"/>
     <addaction name="action_load_from_url"/>
     <addaction name="action_load_from_clipboard"/>
    </widget>
    <widget class="QMenu" name="menu_Save">
     <property name="title">
      <string>儲存 Save</string>
     </property>
     <addaction name="action_save_all"/>
     <addaction name="action_save_selected"/>
    </widget>
    <addaction name="menu_Load"/>
    <addaction name="menu_Save"/>
   </widget>
   <widget class="QMenu" name="menu_preferences">
    <property name="title">
     <string>設定 Preferences</string>
    </property>
    <widget class="QMenu" name="menu_lang">
     <property name="title">
      <string>介面語言 Interface language</string>
     </property>
     <addaction name="action_lang_chi"/>
     <addaction name="action_lang_eng"/>
    </widget>
    <addaction name="menu_lang"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_preferences"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </widget>
  <action name="action_load_from_local">
   <property name="text">
    <string>從本機 From local</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="action_start_detect">
   <property name="text">
    <string>開始 Start</string>
   </property>
  </action>
  <action name="action_load_from_clipboard">
   <property name="text">
    <string>從剪貼簿 From clipboard</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="action_load_from_url">
   <property name="text">
    <string>從地址 From URL</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="action_save_all">
   <property name="text">
    <string>儲存所有處理後的圖片到目錄 Save all processed images to a directory</string>
   </property>
  </action>
  <action name="action_save_selected">
   <property name="text">
    <string>儲存目前有選定的圖片到目錄 Save the selected images to a directory</string>
   </property>
  </action>
  <action name="action_lang_chi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="action_lang_eng">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
